revolv/static/js/base/dashboard.js

```diff
@@ -20,6 +20,73 @@ $(document).ready(function () {
             $this.addClass("fa-rotate-180");
         }
     });
+    
+    /**
+     * This function defines what happens when the different categories are clicked in the
+     * 'my impact' section on the dashboard. The category that is clicked is made active, 
+     * and then an AJAX request is made. The view this AJAX reqest is sent to updates the
+     * preferred_categories attribute of this user.
+     * 
+     */
+    $(".category-option-container").click(function() {
+        $(this).toggleClass("active");
+        var allActiveCategories = $(".category-option-container.active");
+        var allCategories = $(".category-option-container");
+        var dict = {};
+        for (var i = 0; i < allCategories.length; i += 1) {
+            var categoryText = $(allCategories[i]).children().last().text();
```

Comment:
Hm, I guess you're right. Do you think it would be better to use IDs? I think it would be more robust that way, but if you don't think so I don't think it's a big deal